src/styles/blocks/card-block/content-card.scss

```diff
@@ -0,0 +1,134 @@
+.content {
+  margin: 0 auto;
+  display: grid;
+  // gap: 48px;
```

Comment:
The 'gap' CSS property is commented out. If you're not using it, it's better to remove it.